plugins/meta/tuning/tuning.go

```diff
@@ -37,9 +40,12 @@ import (
 	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
 )
 
+const defaultDataDir = "/var/lib/cni/tuning"
```

Comment:
I know we use `/var/lib/cni` for lots of other plugins, but that was a mistake I'm trying to undo. Can you make the default directory `/run/cni/tuning` please?

Reply:
By using `/run`, we don't have to worry about cleaning up stale containers on reboot.

Reply:
Done :)